target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wax50re.c

```diff
@@ -184,6 +187,19 @@ static struct gpio_led tl_wa850re_v2_leds_gpio[] __initdata = {
 	},
 };
 
+static struct gpio_led tl_wa855re_v1_leds_gpio[] __initdata = {
+	 {
+		.name		= "tp-link:green:power",
+		.gpio		= TL_WA855REV1_GPIO_LED_GREEN,
+		.active_low	= 0,
+	},
+	 {
+		.name = "tp-link:red:power",
```

Comment:
Please, be consistent with code. If you are using tabs above, do the same here, don't mix styles.